act-rules-format.html

```diff
@@ -342,6 +342,7 @@ <h2 class="heading settled" data-level="1" id="intro"><span class="secno">1. </s
 		<p>There are currently many test procedures and tools available which aid their users in testing web content for conformance to accessibility standards such as the <a href="https://www.w3.org/WAI/standards-guidelines/wcag/">Web Content Accessibility Guidelines (WCAG)</a> <a data-link-type="biblio" href="#biblio-wcag">[WCAG]</a>. As the web develops in both size and complexity, these procedures and tools are essential for managing the accessibility of resources available on the web.</p>
 		<p>This format is intended to enable a consistent interpretation of how to test for <a href="#accessibility-requirements">accessibility requirements</a> and promote consistent results of accessibility tests. It is intended to be used to describe both manual accessibility tests as well as automated tests performed by accessibility testing tools.</p>
 		<p>Describing how to test certain accessibility requirements will result in accessibility tests that are transparent, with test results that are reproducible. The Accessibility Conformance Testing (ACT) Rules Format defines the requirements for these test descriptions, known as Accessibility Conformance Testing Rules (ACT Rules).</p>
+		<p>Each ACT Rule is a plain English description of what type of content is being tested, the test it performs, and the expected result.  Where the test result affects conformance, the rule documents which particular requirement the test is for. The resulting outcomes from the test can be used to help determine conformance or non-conformance to the requirement. Test cases are also written as part of the ACT rule to provide a way to verify that implementations of the rule can successfully determine the expected pass and fail outcomes.</p>
```

Comment:
This change should be made to the markdown file, not the HTML file. Otherwise we'd lose it.